src/main/java/com/google/codeu/data/Message.java

```diff
@@ -25,20 +25,26 @@
   private String user;
   private String text;
   private long timestamp;
+  private String recipient;
 
   /**
    * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
    * random ID and uses the current system time for the creation time.
    */
-  public Message(String user, String text) {
-    this(UUID.randomUUID(), user, text, System.currentTimeMillis());
+  public Message(String user, String text, String recipient) {
+    this(UUID.randomUUID(), user, text, System.currentTimeMillis(), recipient);
   }
 
-  public Message(UUID id, String user, String text, long timestamp) {
+  /**
+   * Constructs a new {@link Message} posted by {@code user}, of the given 
```

Comment:
Where does recipient fit into this sentence/jsdoc? Same as above basically

Reply:
Also update this method comment to include recipient!